src/osd/PGLog.cc

```diff
@@ -1016,3 +1016,67 @@ void PGLog::rebuild_missing_set_with_deletes(
 
   set_missing_may_contain_deletes();
 }
+
+#ifdef WITH_SEASTAR
+seastar::future<> PGLog::rebuild_missing_set_with_deletes(
```

Comment:
Unless I'm mistaken, this call is only relevant for backwards compatibility.  I'd be fine with replacing the implementation with a ceph_assert(0 == "unsupperted");